### PR TITLE
Fix issue #1 by updating Example to latest Keras API

### PR DESCRIPTION
**Pull request by jasonbunk**
Fix issue #1 by updating the sentiment LSTM in the Example folder to use the latest Keras API. Uses the Theano backend to fix self._predict_stochastic.

Can be tested by inserting `print(str(MC_model_output))` after line 87 of callbacks.py and comparing the difference between `K.learning_phase(): np.uint8(1)` and `K.learning_phase(): np.uint8(0)`
to see the difference between deterministic test-phase dropout and MC dropout.

### CODE REVIEW

**Comment by yaringal** [comment]
Hi @jasonbunk, thanks for the pull request. I can merge it if you want, but I do not intend to maintain this repo in the long term. It's mostly for demonstration purposes. The main code in the repo has been implemented into [Keras](https://keras.io/layers/recurrent/#simplernn), a [TensorFlow](https://github.com/tensorflow/models/blob/4bcbe8e140a6c83c151b93567f33e60e486922bd/neural_programmer/model.py#L106) example, and a [Torch](https://github.com/Element-Research/rnn#gru) package.
